test/dialect/test_sqlite.py

```diff
@@ -280,6 +281,28 @@ def test_no_convert_unicode(self):
             bindproc = t.dialect_impl(dialect).bind_processor(dialect)
             assert not bindproc or isinstance(bindproc("some string"), str)
 
+    def test_bytes(self, connection, metadata):
```

Comment:
**mike bayer** (zzzeek) wrote:

we need this test, or to see that we have one already, in lib/sqlalchemy/testing/suite/test_types.py

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4577

Reply:
```python
class BinaryTest(_LiteralRoundTripFixture, fixtures.TablesTest):
    __requires__ = ("binary_literals",)
    __backend__ = True

    @classmethod
    def define_tables(cls, metadata):
        Table(
            "binary_table",
            metadata,
            Column(
                "id", Integer, primary_key=True, test_needs_autoincrement=True
            ),
            Column("binary_data", LargeBinary),
            Column("pickle_data", PickleType),
        )

    def test_binary_roundtrip(self, connection):
        binary_table = self.tables.binary_table

        connection.execute(
            binary_table.insert(), {"id": 1, "binary_data": b"this is binary"}
        )
        row = connection.execute(select(binary_table.c.binary_data)).first()
        eq_(row, (b"this is binary",))

    def test_pickle_roundtrip(self, connection):
        binary_table = self.tables.binary_table

        connection.execute(
            binary_table.insert(),
            {"id": 1, "pickle_data": {"foo": [1, 2, 3], "bar": "bat"}},
        )
        row = connection.execute(select(binary_table.c.pickle_data)).first()
        eq_(row, ({"foo": [1, 2, 3], "bar": "bat"},))
```

This already exists there.  Was there something else you had in mind to add to it?